src/app/daily/page.tsx

```diff
@@ -39,6 +39,19 @@ export default function DailyLearnSubmitPage() {
         }));
     };
 
+    const handleCategoryChange = (event: { target: { name: any; value: any; }; }) => {
```

Comment:
```suggestion
const handleCategoryChange = (event: { target: { name: string; value: string; }; }) => {
```

지금은 괜찮지만 나중에 서비스 볼륨이 커졌을 때 저희 의도대로 동작하기 위해선 any 사용은 지양하는 편이 좋을 거 같아요!